contrib/database/sql/sql_test.go

```diff
@@ -157,12 +152,11 @@ func TestOpenOptions(t *testing.T) {
 		}
 		sqltest.RunAll(t, testConfig)
 	})
-
 	t.Run("OpenDB", func(t *testing.T) {
-		c, err := pq.NewConnector("postgres://postgres:postgres@127.0.0.1:5432/postgres?sslmode=disable")
-		if err != nil {
-			log.Fatal(err)
-		}
+		Register(driverName, &pq.Driver{}, WithServiceName("postgres-test"), WithAnalyticsRate(0.2))
```

Comment:
I don't see ` WithAnalyticsRate(0.2)` be effectively used, is this necessary?

Reply:
It's used in the assertion from line 175 (also this change is so I keep the test as it was with the `Register` refactor):
```
ext.EventSampleRate: 0.2,
```

Reply:
Ah, sorry